tee-worker/README.md

```diff
@@ -42,6 +42,22 @@ make
 ### Integration Tests
 See [docker/README.md](docker/README.md)
 
+### Quickly launch parachain, tee worker and run ts-test against tee-worker
+
+```
+source "$HOME/.cargo/env"
+source /opt/intel/sgxsdk/environment
```

Comment:
I suggest adding comment that location of this file might differ across environments

Reply:
added the comment now.